oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -0,0 +1,250 @@
+
+Images must be registered to be used by Oracle Cloud services. To register an image in Oracle Cloud, navigate to :guilabel:`Compute` > :guilabel:`Custom Images` and select :guilabel:`Import Image`. 
+
+In the form you must provide your :guilabel:`Compartment`, :guilabel:`Image name`, :guilabel:`Object name` and :guilabel:`Launch mode`. Additionally the fields :guilabel:`Operating System` and :guilabel:`Image type` must be provided and use ``Ubuntu`` and ``QCOW2``, respectively. 
+
+Optionally, this process can be done with the following command:
+
+.. code:: bash
+
+  oci compute image import from-object \
+    --compartment-id <compartment-id> \
+    --display-name <image-name> \
+    --launch-mode <launch-mode> \
+    --image-source-object-name <object-name> \
+    --bucket-name <bucket-name> \
+    --operating-system "Ubuntu" \
+    --operating-system-version <ubuntu-version-number> \
+    --source-image-type QCOW2 \
+    --wait-for-state AVAILABLE
+
+
+Creating OKE Nodes with Ubuntu Images
+-------------------------------------
+
+The following steps on creating nodes assumes you have an existing OKE cluster on Oracle Cloud, but it is not required to have existing nodes. If you don't have an OKE cluster prepared then Oracle's documentation for `creating a cluster`_ is a good place to start.
+
+Creating Managed OKE nodes with Ubuntu
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Managed nodes are node instances whose lifecycle is managed by the OKE service. Since this is a Limited Availability release of Ubuntu images for OKE, you can only create managed nodes through the Oracle Cloud API (``oci`` CLI or SDK). The ability to create managed from nodes from the Oracle Cloud UI will be added at a later.
+
+To create a managed node, start by copying the following cloud-init script into a file called ``user-data.yaml``.
+
+.. code:: yaml
+
+   #cloud-config
+   
+   runcmd:
+     - oke bootstrap
+
+Then, create a placement configuration file to specify where in Oracle Cloud the Managed node pool should be created and save the file as ``placement-config.json``.
+
+.. code:: json 
+
+   [{
+     "compartmentId":"<compartment-id>",
+     "availabilityDomain":"<availability-domain>",
+     "subnetId":"<subnet-id>"
+   }]
+
+
+Lastly, replace the values and run the following command to create the Managed node pool:
+
+.. code:: bash
+   
+  oci ce node-pool create \
+    --cluster-id=<cluster-id> \
+    --compartment-id=<compartment-id> \
+    --name=<pool-name> \
+    --node-shape=<node-shape> \
+    --size=<pool-count> \
+    --kubernetes-version="1.29.1" \
+    --node-image-id=<ubuntu-image-id> \
+    --placement-configs="$(cat placement-config.json)" \
+    --node-metadata='{"user_data": "'"$(base64 user-data.yaml)"'"}'
+
+
+View the node pool status in Oracle Cloud by navigating to :guilabel:`Kubernetes Clusters (OKE)` and click on your cluster, then select :guilabel:`Resources` > :guilabel:`Node pools` and click on the latest node pool.
```

Comment:
Try to use 'select' instead of 'click' whenever possible. (Since people might not necessarily be using a mouse)